common/models/user.js

```diff
@@ -83,11 +83,6 @@ function getWaitPeriod(ttl) {
   return 0;
 }
 module.exports = function(User) {
-  // NOTE(berks): user email validation currently not needed but build in. This
-  // work around should let us sneak by
-  // see:
-  // https://github.com/strongloop/loopback/issues/1137#issuecomment-109200135
-  delete User.validations.email;
```

Comment:
We will need to keep this in mind as not everyone has an email address listed. For users who exclusively sign in using github may have issues here. I think we can move forward with this and watch our final QA for issues with Github Oauth.

Reply:
Thanks I have created a new issue to track this. https://github.com/freeCodeCamp/freeCodeCamp/issues/16126